scripts/guest/m-reinstall

```diff
@@ -74,7 +74,7 @@ install_cmd="./bin/magento setup:install \
     --use-rewrites=1"
 
 # Configure Rabbit MQ
-if [[ -f "${MAGENTO_ROOT}/LICENSE_EE.txt" ]]; then
+if [[ -f "${MAGENTO_ROOT}/app/code/Magento/MessageQueue/LICENSE.txt" ]]; then
```

Comment:
This will fail for composer-based installation because all Magento modules are located in vendor in that case.